pkgs/development/libraries/embree/default.nix

```
@@ -24,8 +24,14 @@ stdenv.mkDerivation rec {
     "-DEMBREE_RAY_MASK=ON"
   ];
 
+  patchPhase = ''
+    substituteInPlace common/math/math.h --replace 'defined(__MACOSX__) && !defined(__INTEL_COMPILER)' 0
+    substituteInPlace common/math/math.h --replace 'defined(__WIN32__) || defined(__FreeBSD__)' 1
```

```suggestion
    substituteInPlace common/math/math.h --replace 'defined(__WIN32__) || defined(__FreeBSD__)' 'defined(__WIN32__) || defined(__FreeBSD__) || defined(__MACOSX__)'
```

We have `sincosf` on linux.